mitodl/open-discussions
# remove anon access flag

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #1763 

#### What's this PR do?

This PR removes the anonymous access feature flag on both the frontend and the backend, so that the anonymous access behavior is now the default behavior.

As part of this I deleted a large number of cassettes which were generated for previously-existing parameterized test cases where the feature flag was turned off.

#### How should this be manually tested?

This touches a lot of parts of the app, so the important thing is to make sure that all functions of the app are working. Anonymous access to channels which are set to public should be working well, anonymous users shouldn't be able to access private content, etc.
